dbt/include/snowflake/macros/relations/table/create.sql

```diff
@@ -1,6 +1,15 @@
 {% macro snowflake__create_table_as(temporary, relation, compiled_code, language='sql') -%}
+  {%- set transient = config.get('transient', default=true) -%}
+
+  {% if temporary -%}
+    {%- set table_type = "temporary"
```

Comment:
All of these `{%- set` need a `%}` at the end

Reply:
thank you ❤️